x-pack/plugins/alerting/server/routes/unmute_all_rule.ts

```diff
@@ -29,6 +33,11 @@ export const unmuteAllRuleRoute = (
       },
       validate: {
         params: paramSchema,
+        response: {
+          204: {
+            description: 'Indicates a successful call.',
+          },
+        },
```

Comment:
```suggestion
        request: {
          params: paramSchema,
        },
        response: {
          204: {
            description: 'Indicates a successful call.',
          },
        },
```

Reply:
Good catch @adcoelho , I've created [this PR](https://github.com/elastic/kibana/pull/190300) that should help TS catch this for us in future